<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="fromAtoBs-MBP" failures="0" tests="5" name="com.atob.qa.tests.HomePageTest" time="0.027" errors="1" timestamp="05 Sep 2019 07:12:22 GMT" skipped="4">
  <testcase classname="com.atob.qa.tests.HomePageTest" name="setUp" time="0.027">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at org.openqa.selenium.support.events.EventFiringWebDriver.extractInterfaces(EventFiringWebDriver.java:120)
at org.openqa.selenium.support.events.EventFiringWebDriver.<init>(EventFiringWebDriver.java:94)
at com.atob.qa.base.TestBase.initialization(TestBase.java:60)
at com.atob.qa.tests.HomePageTest.setUp(HomePageTest.java:37)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- setUp -->
  <testcase classname="com.atob.qa.tests.HomePageTest" name="homePageTitleTest" time="0.000">
    <skipped/>
  </testcase> <!-- homePageTitleTest -->
  <testcase classname="com.atob.qa.tests.HomePageTest" name="homePageHeadingTest" time="0.000">
    <skipped/>
  </testcase> <!-- homePageHeadingTest -->
  <testcase classname="com.atob.qa.tests.HomePageTest" name="selectBahnCard" time="0.000">
    <skipped/>
  </testcase> <!-- selectBahnCard -->
  <testcase classname="com.atob.qa.tests.HomePageTest" name="searchATrip" time="0.000">
    <skipped/>
  </testcase> <!-- searchATrip -->
</testsuite> <!-- com.atob.qa.tests.HomePageTest -->
